package.json

```diff
@@ -40,7 +41,9 @@
     "react-dom": "16.8.6",
     "redux": "^4.0.1",
     "redux-saga": "^1.0.2",
-    "sass-loader": "^7.1.0"
+    "sass-loader": "^7.1.0",
+    "session-file-store": "^1.2.0",
```

Comment:
Since session-file-store isn't used anymore, remove it

Reply:
I will actually prefer will give to as session-file, I changed to Mongo Store because of the bugs we were having. 
@enyeInc what do you advise we go for, a file store or a db store for sessions relating this to your advice on reducing db request?